src/main/java/com/denizenscript/depenizen/bukkit/events/crackshot/crackshotPlayerFinishesReloadingWeaponEvent.java

```diff
@@ -0,0 +1,73 @@
+package com.denizenscript.depenizen.bukkit.events.crackshot;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.shampaggon.crackshot.events.WeaponReloadCompleteEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class crackshotPlayerFinishesReloadingWeaponEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // crackshot player starts reloading weapon
+    //
+    // @Regex ^on crackshot player starts reloading weapon$
+    //
+    // @Triggers when a player starts reloading their CrackShot weapon.
+    //
+    // @Cancellable false
+    //
+    // @Context
+    // <context.weapon_name> returns the name of the weapon.
```

Comment:
Why is this `weapon_name` but the other event has `weapon`?

Reply:
Some events return an ItemStack, others return a weapon name.. I just used the available methods.